java/add-point-scene-layer/README.md

```diff
@@ -0,0 +1,37 @@
+# Add point scene layer
+
+View a point scene layer from a scene service.
+
+![Add point scene layer App](add-point-scene-layer.png)
+
+## Use case
+
+Point scene layers can efficiently display large amounts of point features. While point cloud layers can only display simple symbols, point scene layers can display any type of billboard symbol or even 3D models, as long as the location of the symbol can be described by a point. Points are cached and automatically thinned when zoomed out to improve performance.
+
+## How to use the sample
+
+Pan around the scene and zoom in. Notice how many thousands of additional features appear at each successive zoom scale.
+
+## How it works
+
+1. Create a scene.
+2. Create an `ArcGISSceneLayer` with the URL to a point scene layer service.
+3. Add the layer to the scene's operational layers collection.
```

Comment:
Do you want to add a step for creating the service and using the world elevation?

Reply:
Not critical to the construction of a point scene layer (just makes it look better), so going to leave it out.